packages/react-components/react-progress/src/components/ProgressBar/ProgressBar.test.tsx

```diff
@@ -47,4 +53,27 @@ describe('ProgressBar', () => {
     expect(result.getByRole('progressbar').getAttribute('aria-valuemin')).toEqual('0');
     expect(result.getByRole('progressbar').getAttribute('aria-valuemax')).toEqual('1');
   });
+  describe('Max', () => {
+    it('gives the proper error message when max is negative', () => {
+      const max = -1;
+      const errorMsg = `The prop 'max' must be greater than 0. Received max: ${max}`;
+      render(<ProgressBar max={max} />);
+      expect(console.error).toHaveBeenCalledWith(errorMsg);
+    });
+    it('gives the proper error message when max is zero', () => {
+      const max = 0;
+      const errorMsg = `The prop 'max' must be greater than 0. Received max: ${max}`;
+      render(<ProgressBar max={max} />);
+      expect(console.error).toHaveBeenCalledWith(errorMsg);
+    });
+  });
+  describe('Value', () => {
+    it('gives the proper error message when value is greater than max', () => {
```

Comment:
Ditto to above, let's add a simple test to verify no error is logged when everything is valid.